packages/expo-sqlite/src/SQLiteDatabase.ts

```diff
@@ -81,7 +81,7 @@ export class SQLiteDatabase {
    *   // The following UPDATE query out of transaction may be executed here and break the expectation.
    *   //
    *
-   *   const result = await db.getAsync<{ name: string }>('SELECT name FROM Users');
+   *   const result = await db.getAllAsync<{ name: string }>('SELECT name FROM Users');
```

Comment:
```suggestion
   *   const result = await db.getFirstAsync<{ name: string }>('SELECT name FROM Users');
```

since we have `result?.name`, it should be getFirstAsync() for single entitle.

Reply:
Thanks for correcting that! Let me update this!

Reply:
Looks like a couple of the tests are broken @Kudo. Unrelated to this change. They are already failing on main

Reply:
> since we have result?.name, it should be getFirstAsync() for single entitle.

Done.